Sources/Swift/FlexLayout.swift

```diff
@@ -43,6 +43,19 @@ public final class Flex {
     public var intrinsicSize: CGSize {
         return yoga.intrinsicSize
     }
+
+    /**
+     This`direction` property is get the direction flex items are placed in the flex container.
+     */
+    public var direction: Direction? {
+        switch yoga.flexDirection {
+        case .column:           return Flex.Direction.column
+        case .columnReverse:    return Flex.Direction.columnReverse
+        case .row:              return Flex.Direction.row
+        case .rowReverse:       return Flex.Direction.rowReverse
+        @unknown default:       return nil
+        }
```

Comment:
1. Could you do something like `isIncludedInLayout` where we can get & set the value using the property. 
2. Could you move the property definition just above the `direction(_ value: Direction)` method definition.
3. You can copy the documentation of `direction(_ value: Direction)` for the property. For the README also.
Thanks 

```
public var direction: Direction? {
        get {
            switch yoga.flexDirection {
                case .column:           return Flex.Direction.column
                case .columnReverse:    return Flex.Direction.columnReverse
                case .row:              return Flex.Direction.row
                case .rowReverse:       return Flex.Direction.rowReverse
                @unknown default:       return nil
             }
        }
        set {
            direction(newValue)
        }
    }
```

Reply:
@lucdion 
1. Of course. I did that at first, but I changed it after looking at the intrinsicSize property.
2., 3. Got it!
Thanks for review

[8cdf133](https://github.com/layoutBox/FlexLayout/pull/213/commits/8cdf1336e522c47a9f386c317018181e9b78d46c)